autoload/ack.vim

```diff
@@ -2,14 +2,6 @@ if exists('g:autoloaded_ack') || &cp
   finish
 endif
 
-if exists('g:ack_use_dispatch')
-  if g:ack_use_dispatch && !exists(':Dispatch')
-    call s:Warn('Dispatch not loaded! Falling back to g:ack_use_dispatch = 0.')
```

Comment:
This was failing before as `s:Warn` was defined only at the end of the script, so I figured we could move this (and the additional check I added for async) inside `s:Init()`